data_steward/tools/load_vocab.py

```diff
@@ -223,6 +292,14 @@ def get_arg_parser() -> argparse.ArgumentParser:
         help=
         'Identifies the target dataset where the vocabulary is to be loaded',
         required=False)
+    argument_parser.add_argument(
+        '-s',
+        '--staging_dataset_id',
+        dest='staging_dataset_id',
+        action='store',
+        help=
+        'Identifies the staging dataset where the vocabulary is to be staged',
+        required=False)
```

Comment:
This argument is never used. Is it a placeholder?

Reply:
Good point. I was wondering if the user should supply this or the scripts should infer it. I'll remove this since the script is already inferring the name.

Reply:
Sounds good